ce_deploy.module

```diff
@@ -107,3 +107,72 @@ function ce_deploy_create_taxonomy($filename) {
     }
   }
 }
+
+/**
+ * Ensures icon files for ckeditor embed buttons are available.
+ *
+ * WHY: Embed icons are non-exportable. There's a pending patch in d.org to make
+ * it possible to reference them by actual path, and not only file_managed
+ * (uuid). For the time being, custom solution is used.
+ *
+ * This function does 2 things:
+ *
+ * 	1.- Scans the available files for the buttons specified in $buttons_map, and
+ *  moves them to the directory where the embed module expects them. The files
+ *  are meant to be in the "images/embed_icons" of the ce_deploy module.
+ *
+ * 	2.- Then loads the available embed buttons, and checks the ones specified
+ *  in the $button_map array. For those, checks if there's an existinf file. If
+ *  there is, it does nothing. If there isn't, it picks the specified file from
+ *  the map array, creates a File entity in the database, and assigns to it the
+ *  icon uuid expected by the button entity.
+ *
+ * @link https://www.drupal.org/project/embed/issues/2824110.
+ */
+function ce_deploy_ensure_embed_buttons_icons() {
+	$buttons_map = [
```

Comment:
Can we make buttons_map an argument to function ? so you can call it multiple time in different hook_update_n.